k8s/mongodb/container/configure_mdb_users.template.js

```diff
@@ -0,0 +1,43 @@
+use admin;
```

Comment:
Does this work? I was under the impression that the JavaScript equivalent of `use admin` (a mongo shell command) is

```javascript
db = db.getSiblingDB("admin")
```

Reply:
Let me try this then I can update.